include/sparrow/layout/list_layout/list_array.hpp

```diff
@@ -491,4 +502,44 @@ namespace sparrow
         const auto offset = i * m_list_size;
         return std::make_pair(offset, offset + m_list_size);
     }
+
+
+    inline arrow_proxy fixed_sized_list_array::create_proxy(std::uint64_t list_size, array && flat_values)
```

Comment:
I think we should allow to pass a range for the null values (similar to what we do [here](https://github.com/man-group/sparrow/blob/main/test/external_array_data_creation.cpp#L113-L121))

Reply:
agree!